flink-runtime/src/test/java/org/apache/flink/runtime/jobmaster/JobMasterTest.java

```diff
@@ -2022,17 +2026,18 @@ private File createSavepointWithOperatorState(long savepointId, OperatorID... op
 	}
 
 	private Collection<OperatorState> createOperatorState(OperatorID... operatorIds) {
+		Random random = new Random(0x42);
```

Comment:
The values from this random don't have to be globally unique. So it doesn't matter if it's the same seed or not for different calls. 
However, if this method would be used differently, it could matter. 
So I'll remove the seed.